tests/task_comments/test_api_report_violation_on_task_comment.py

```diff
@@ -0,0 +1,67 @@
+import json
+import unittest
+
+from flask_restx import marshal
+
+from app import messages
+from app.api.dao.task_comment import TaskCommentDAO
+from app.api.models.mentorship_relation import task_comments_model
+from tests.tasks.tasks_base_setup import TasksBaseTestCase
+from tests.test_utils import get_test_request_header
+
+class TestReportViolationOnTaskCommentAPI(TasksBaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.relation_id = self.mentorship_relation_w_admin_user.id
+        self.task_id = 1
+
+        TaskCommentDAO().create_task_comment(
+            user_id=1, task_id=1, relation_id=self.relation_id, comment="comment"
+        )
+
+    def test_report_violation_without_auth_header(self):
+        # Call API without header
+        expected_response = messages.AUTHORISATION_TOKEN_IS_MISSING
+        actual_response = self.client.post(
+            f"mentorship_relation/{self.relation_id}/task/{self.task_id}/comment/{1}/report",
```

Comment:
You are using both numbering and variables to format the string. Can we use just one way?

Reply:
Done. Literals replaced with variables.